demo.js

```diff
@@ -123,6 +123,18 @@
   var defaultBackground = palettes.style.backgroundColor;
   var tooltip = byId('tooltip');
 
+  function hex(x) {
+    return ("0" + parseInt(x).toString(16)).slice(-2);
```

Comment:
The rest of the file uses single quotes for strings so best to keep consistent here as well.

Reply:
Sure